x-pack/plugins/observability_ai_assistant/server/functions/esql/docs/esql-row.txt

```diff
@@ -1,17 +1,35 @@
 ROW
 
 Syntax
+```esql
 ROW column1 = value1[, ..., columnN = valueN]
+```
+
 Parameters
 columnX
 The column name.
 valueX
 The value for the column. Can be a literal, an expression, or a
 function.
-DescriptionThe ROW source command produces a row with one or more columns with values
+DescriptionThe
```

Comment:
Is this ok?

Reply:
Yeah, the LLM processes it as separate tokens anyway.

Reply:
I would personally put this on separate lines as it looks buggy for humans, but will leave to your judgement.

Reply:
Sorry, I should have clarified, these documents are automatically extracted from the ES|QL documentation in `@elastic/built-docs`. I want to avoid making manual adjustments (and putting too much effort in the scripting part if it does not make a difference to the LLM).